libs/withdraws/src/lib/withdraw-form.tsx

```diff
@@ -197,10 +196,10 @@ export const WithdrawForm = ({
         data-testid="withdraw-form"
       >
         <TradingFormGroup
-          label={t('To ({{chainName}} address)', {
+          label={t('To {{chainName}} address', {
             chainName: selectedAsset?.chainId
               ? getChainName(selectedAsset.chainId)
-              : ETHEREUM_CHAIN_ID,
+              : '',
```

Comment:
**nitpick (non-blocking)**: Again, better then blank would be showing the chainId. Again again, not a blocker.

Reply:
In this case there `selectedAsset.chainId` is falsey so we don't have a chainId on line 202. I would rather it fall back to just 'To address' or 'From address', rather than defaulting to '1'